drake/solvers/MultipleTreeProblem.m

```diff
@@ -0,0 +1,463 @@
+classdef MultipleTreeProblem
+  
+  properties
+    robot
+    endEffectorId
+    xStart
+    xGoal
+    endEffectorPoint
+    goalConstraints
+    additionalConstraints
+    qNom
+    mergingThreshold
+    capabilityMap
+    graspingHand
```

Comment:
I have now separated:
* the MultipleTreeProblem: multiple rrt(*) tree solver
* from FinalPoseProblem: which uses a capability map to find the goal configuration